ts/Dashboards/Components/Component.ts

```diff
@@ -45,6 +44,7 @@ const {
 } = DG;
 import DataTable from '../../Data/DataTable.js';
 import EditableOptions from './EditableOptions.js';
+import Globals from '../Globals.js';
```

Comment:
Is it necessary to import the default twice from `'../Globals.js'` as `Globals` and `DG`? I guess the `DG` can be removed and `classNamePrefix` can be taken from `Globals`. I mean like:
```ts
import Globals from '../Globals.js';
const {
    classNamePrefix
} = Globals;
```